source/includes/mongodb-compatibility-table-go.rst

```diff
@@ -4,14 +4,25 @@
    :class: compatibility-large
 
    * - Go Driver
+     - MongoDB 4.2
      - MongoDB 4.0
      - MongoDB 3.6
      - MongoDB 3.4
      - MongoDB 3.2
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 1.1.0
```

Comment:
I suggest dropping trailing ".0" here as well, and any other places where a placeholder for the patch version is used (be it ".0" or ".x"). Patch versions should be irrelevant as far as server compatibility is concerned.